src/Submission/ProjectSubmission.jsx

```diff
@@ -81,8 +81,9 @@ class ProjectSubmission extends React.Component {
         !useArboristUI
         || (isRootUrl(project) && userHasSheepdogProgramAdmin(userAuthMapping))
         || (isProgramUrl(project) && userHasSheepdogProjectAdmin(userAuthMapping))
-        || userHasMethodOnProject('create', project, userAuthMapping)
-        || userHasMethodOnProject('update', project, userAuthMapping)
+        || userHasMethodForServiceOnProject('create', '*', project, userAuthMapping)
+        || userHasMethodForServiceOnProject('update', '*', project, userAuthMapping)
+        || userHasMethodForServiceOnProject('*', '*', project, userAuthMapping)
```

Comment:
you shouldn't need this line if `userHasMethodForServiceOnResource` is fixed to handle `*` properly

Reply:
Oh this is actually a question i wanted to ask, I'm thinking do we need to check for `{service: sheepdog, method: *}`?

Right now Portal checks for `/services/sheepdog/submission/project` and `/services/sheepdog/submission/program` to see if user has sheepdog admin, could we use `{service: sheepdog, method: *}` instead?

Reply:
> do we need to check for `{service: sheepdog, method: *}`?

ugh, yeah maybe 🤔 and for the download button it should be `fence`, etc

> could we use `{service: sheepdog, method: *}` instead?

if by "sheepdog admin" you mean program/project creation, then `/services/sheepdog/submission/project` and `/services/sheepdog/submission/program` are right. But if you mean "access to all the data", then `{service: sheepdog, method: *}`